src/torrent.cpp

```diff
@@ -11579,6 +11579,7 @@ namespace libtorrent
 	{
 		if (!m_apply_ip_filter) return;
 		if (!m_peer_list) return;
+		if (!m_ip_filter) return;
```

Comment:
oh, right.. it dereferences the ip_filter.. I guess that's always expected to be set then. a bit weird..